deploy/examples/operator.yaml

```diff
@@ -589,6 +589,17 @@ spec:
             - name: DISCOVER_DAEMON_UDEV_BLACKLIST
               value: "(?i)dm-[0-9]+,(?i)rbd[0-9]+,(?i)nbd[0-9]+"
 
+            - name: DISCOVER_DAEMON_RESOURCES
```

Comment:
the same change is required in helm charts, helm documentation, and operator-openshift.yaml

Reply:
Done